pygen/pygen_src/riscv_asm_program_gen.py

```diff
@@ -596,17 +596,17 @@ def get_directed_instr_stream(self):
             arg = "directed_instr_{}".format(i)
             stream_name_opts = "stream_name_{}".format(i)
             stream_freq_opts = "stream_freq_{}".format(i)
-            if(arg in args):
-                val = args_dict[arg]
+            if(arg in cfg.argv):
```

Comment:
I think you can iterate over `cfg.args_dict` directly, iterating over a `Namespace` object will result in errors.